src/components/SeeResults/SeeResults.tsx

```diff
@@ -384,32 +391,260 @@ function SeeResults(): JSX.Element {
-					changeFrame(`next`)
-				}}
-			>
-				next
-			</button>
+			<main>
+				<header>{resultsView.round}</header>
+				<main>
+					<LayoutGroup>
+						{electionRef.current ? (
+							<Phase election={electionRef.current}>
+								<Keyframe state={state} election={electionRef.current} />
+							</Phase>
+						) : (
+							<div data-keyframe="loading">
+								<header>loading</header>
+							</div>
+						)}
+					</LayoutGroup>
+				</main>
+			</main>
+			<nav>
+				<button
+					type="button"
+					disabled={viewLocation === `beginning`}
+					onClick={() => {
+						changeFrame(`prev`)
+					}}
+				>
+					prev
+				</button>
+				<button
+					type="button"
+					disabled={viewLocation === `end`}
+					onClick={() => {
+						changeFrame(`next`)
+					}}
+				>
+					next
+				</button>
+			</nav>
+			<aside>
+				{electionRef.current ? <ElectionRounds election={electionRef.current} /> : null}
+			</aside>
 		</div>
 	)
 }
 
+export const Phases = {
+	surplus_allocation({ children }) {
+		return (
+			<div data-phase="surplus_allocation">
+				<header>surplus allocation</header>
+				<main>{children}</main>
+			</div>
+		)
+	},
+	winner_selection({ children, election }) {
+		const roundsLength = useO(election.state.roundsLength)
+		return (
+			<motion.div layoutId="phase" data-phase="winner_selection">
+				<header>winner selection</header>
+				<main>{roundsLength > 0 ? children : null}</main>
+			</motion.div>
+		)
+	},
+	loser_selection({ children }) {
+		return (
+			<motion.div layoutId="phase" data-phase="loser_selection">
+				<header>loser selection</header>
+				<main>{children}</main>
+			</motion.div>
+		)
+	},
+	done({ children }) {
+		return (
+			<motion.div layoutId="phase" data-phase="done">
+				<header>done</header>
+				<main>{children}</main>
+			</motion.div>
+		)
+	},
+} satisfies Record<
+	ResultsViewPhase,
+	FunctionComponent<{ children: ReactNode; election: ElectionInstance }>
+>
+
+export const Keyframes = {
+	// surplus_allocation
+	show_surplus_ratio(_) {
+		return (
+			<motion.div layoutId="keyframe" data-keyframe="show_surplus_ratio">
+				<header>show surplus ratio</header>
+			</motion.div>
+		)
+	},
+	show_alternative_consensus(_) {
+		return (
+			<motion.div layoutId="keyframe" data-keyframe="show_alternative_consensus">
+				<header>show alternative consensus</header>
+			</motion.div>
+		)
+	},
+	compress_alternative_consensus(_) {
+		return (
+			<motion.div layoutId="keyframe" data-keyframe="compress_alternative_consensus">
+				<header>compress alternative consensus</header>
+			</motion.div>
+		)
+	},
+	distribute_surplus(_) {
+		return (
+			<motion.div layoutId="keyframe" data-keyframe="distribute_surplus">
+				<header>distribute surplus</header>
+			</motion.div>
+		)
+	},
+	// winner_selection
+	show_candidates({ election }) {
+		const view = useO(resultsViewAtom)
+		const currentRound = election.rounds[view.round]
+		// biome-ignore lint/style/noNonNullAssertion: we swag this
```

Comment:
`swag` ?

Reply:
haha sorry just my default comment when I'm working through a temporary thing. should all be changed now